corehq/apps/app_manager/urls.py

```diff
@@ -128,6 +129,7 @@
     url(r'^download/(?P<app_id>[\w-]+)/$', 'download_index', name='download_index'),
     url(r'^download/(?P<app_id>[\w-]+)/CommCare.ccz$', DownloadCCZ.as_view(),
         name=DownloadCCZ.name),
+    url(r'^download/(?P<app_id>[\w-]+)/multimedia/', include(media_download_urls)),
```

Comment:
might be good to just add a comment in the code explaining that so no one changes the order in the future